docs/content/quickstart/parameters.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "QuickStart: Parameters"
+descriptions: Learn how to use a bundle with parameters
+layout: single
+---
+
+Now that you know how to install a bundle, let's look at how to specify parameters to customize how that bundle is installed.
+Bundles can define parameters to allow the end-user to tweak how the bundle is configured.
```

Comment:
Needs to be active here. 

Bundle authors add parameters to bundles so that end-users can tweak how the bundle is configured at installation. An example of where this would be useful is supplying a username and password value for a database.

Reply:
I am going to leave off the second sentence because a) those are most likely credentials and I don't want to confuse the issue of creds vs params just yet, and b) there are 3 examples of parameters in the next paragraph.